### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by emmycn**
This is the initial commit for this repository

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("hello world")
```
Comment (bryan-brilivan) [inline script.js:1]: what is this for?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (bryan-brilivan) [inline index.html:6]: Why is it called document?

**Review by bryan-brilivan — COMMENTED** [review]
Please check comments

File: script.js (hunk 1 above)
Comment (emmycn) [reply script.js:1]: this is just a test

File: index.html (hunk 2 above)
Comment (emmycn) [reply index.html:6]: Because it is

**Review by bryan-brilivan — APPROVED** [review]
LGTM

**Pull request merged**